package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@descript/superpowered",
   "description": "Superpowered interactive audio features in JavaScript + WebAssembly.",
-  "version": "2.2.0-rc16",
+  "version": "2.2.0-rc17",
```

Comment:
Why the version bump?

Reply:
Bumping the version so we can publish the changes in this PR and consume it in `descript-web-v2`